scripts/e2e_setup_cluster.sh

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
```

Comment:
Also here, the user shell gets closed if an error occurs

Reply:
Done.